controllers/users.js

```diff
@@ -75,6 +75,34 @@ const getUsers = async (req, res) => {
   }
 };
 
+/**
+ * Returns the usernames of inactive/idle users/members
+ *
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ */
+
+const getIdleUsers = async (req, res) => {
+  try {
+    const allUsers = await userQuery.fetchUsers(req.query);
```

Comment:
Here we are fetching all users from firebase and then filtering at our application level. Let's say if we have 1,00,000 users and just 10 idle users then we'll be charged by firebase for reading 1,00,000 user documents when we wanted to use only 10 of them. 
We should specify the filtration criteria in firebase query

This might help: https://github.com/Real-Dev-Squad/website-backend/pull/649#discussion_r935730449

Reply:
This PR will be dependent on issue https://github.com/Real-Dev-Squad/website-backend/issues/754